tensorflow/compiler/tests/image_ops_test.py

```diff
@@ -514,6 +514,24 @@ def testAlignCorners3x3To12x12(self):
                            [7, 7, 7, 8, 8, 8, 8, 8, 8, 9, 9, 9]],
                           dtype=np.float32))
 
+  def testAlignCorners3x3To12x12_uint8(self):
```

Comment:
Duplicate the previous test point, testAlignCorners3x3To12x12, and change data type from float32 to uint8.